services/dynamic-scheduler/requirements/_test.in

```diff
@@ -16,6 +16,7 @@ coverage
 docker
 faker
 pytest
+pytest-aiohttp
```

Comment:
as far as I know, that package is replaced by pytest-asyncio completely and was deprecated. Please check that and remove it.